src/Connector.sol

```diff
@@ -51,9 +63,7 @@ contract CentrifugeConnector {
     event TrancheAdded(uint256 indexed poolId, bytes16 indexed trancheId);
     event TrancheDeployed(uint256 indexed poolId, bytes16 indexed trancheId, address indexed token);
 
-    constructor(address escrow_, address tokenFactory_, address memberlistFactory_) {
-        escrow = EscrowLike(escrow_);
-
+    constructor(address tokenFactory_, address memberlistFactory_) {
```

Comment:
I think we should keep `escrow` in the constructor and not make it fileable. The escrow contract should never have to be swapped (as this would require token transfers for every holder).

Reply:
Good catch ☝️